README.md

```diff
@@ -57,3 +61,16 @@ a test user. This is a one-time task to run only once when setting up your
 development environment.
 
 Use the `npm test` command to run the tests.
+
+## Module Long Term Support Policy
+
+This module adopts the [Module Long Term Support (LTS)](http://github.com/CloudNativeJS/ModuleLTS) policy, with the following End Of Life (EOL) dates:
+
+| Version | Status          | Published | EOL      |
+| ------- | --------------- | --------- | -------- |
+| 4.x     | Active LTS      | Sep 2017  | Dec 2019 |
+| 3.x     | Maintenance LTS | Jul 2014  | Apr 2019 |
+| 2.x     | Maintenance LTS | Nov 2013  | Apr 2019 |
```

Comment:
The older Node.js version in LTS is 6.x, it entered LTS mode on 2016-10-18. At that time, the latest version of loopback-workspace was 3.x and that's the older version of loopback-workspace we need to support.

No need to support 2.x & 1.x versions any more, please remove these two lines.